libs/community/langchain_community/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -121,6 +121,9 @@ def __init__(
         self._embedding_key = self._vector_embedding_policy["vectorEmbeddings"][0][
             "path"
         ][1:]
+        self._distance_strategy = self._vector_embedding_policy["vectorEmbeddings"][0][
+            "distanceFunction"
```

Comment:
is this guaranteed to always be specified? seems like we only check that self._vector_embedding_policy isn't empty if self._create_container is True

Reply:
Actually, this assumption is not always valid 
> Once created, it's possible to query for these info in the properties of the container. (through container.read())

It seems there is a bug in azure cosmos db no sql but if the container was the result of a partition_key migration then it seems the information on the vector policy is lost (so not possible to read through `container.read()`). However, the container still fully supports vector search !

I'll readapt the implementation to account for this case.

I'm sorry for the long post I wanted to be thorooughly descriptive and argumentative in my implementation choices!